File: 100-Days-Series/Day 80/MultiplicationTable.js

// Arrow Function to print the multiplication table for a given number.
const getPrintTable=(number)=>{
    console.log("Multiplication table for "+number+" :");
    for(let i=1;i<=10;i++){
        //Correcting the multiplication logic in the loop
        console.log(number+" x "+i+" = "+(number*i));
    }
}

// Let's Start Here
let number=10;
// Call the function to print its multiplication table
getPrintTable(number);

File: 100-Days-Series/Day 80/MultiplicationTable.c
#include<stdio.h>
// Function to print the multiplication table for a given number
void getPrintTable(int number) {
    printf("\nMultiplication Table for %d:", number);
    for (int i = 1; i <= 10; i++) {
        // Correcting the multiplication logic in the loop
        printf("\n%d x %d = %d", number, i, (number * i));
    }
}
// Let's Start Here
void main() {
    int number;
    // Prompt the user to enter a number
    printf("\nEnter a number to print its multiplication table: ");
    scanf("%d", &number);
    // Call the function to print the multiplication table
    getPrintTable(number);
}
